lib/sairediscommon.h

```diff
@@ -67,6 +67,11 @@
  */
 #define REDIS_TABLE_NOTIFICATIONS   "NOTIFICATIONS"
 
+/**
+ * @brief Table which will be used to forward notifications from gbsyncd.
+ */
+#define REDIS_TABLE_GBNOTIFICATIONS "GBNOTIFICATIONS"
```

Comment:
why GB needs different communication channel? channels are based on databases, for example ASIC_DB, so if GB will have different database then it will have separate communication channel for notifications and they will not colide

Reply:
also this could be generalizaed by chaning notifications channel name by prefixing it by db name
for example m_dbASIC + REDIS_TABLE_NOTIFICATIONS, then it would be more generic, and i  think this would be proffered way to make this change

Reply:
The redis channel is global per instance. Do you prefer to the below?

```
#define REDIS_GB_ASIC_NOTIFICATIONS "GB_ASIC_NOTIFICATIONS"
```

"NOTIFICATIONS" has been widely referred as asic channel. I'll not rename it.

Reply:
i think more preffered way would be to use that asic db prefix

Reply:
Add `REDIS_TABLE_NOTIFICATIONS_PER_DB(dbName)`